nilearn/datasets/struct.py

```diff
@@ -122,40 +122,53 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
-def load_mni152_template():
-    """Load the skullstripped 1mm-resolution version of the MNI152 T1 template
-    originally distributed with FSL.
+def load_mni152_template(resolution=1):
+    """Load the skullstripped 1mm-resolution version of the MNI152 T1 template,
```

Comment:
I think we need to set the default value to 2mm for backward compatibility reasons (previous code using `load_mni152_template()` and expecting 2mm will still get the expected output). 
We can also add a warning saying that the default resolution will change to 1mm in release 0.9 if we are planning to do so.

Reply:
I changed the default resolution to 1mm but I didn't add the warning message. I was wondering whether we shouldn't add a deprecation to these 3 load template functions, instead. Yet, in that case, I think we need to rename these functions, as well.